ipmof/parameters.py

```diff
@@ -14,6 +14,7 @@
                 'grid_size': 1,                  # Grid size for potential energy map (Angstrom)
                 'force_field': 'uff',            # Force field selection for LJ ('uff' or 'dre')
                 'core_database': False,          # Use CoRE database information or not
+                'energy_map_type': 'numpy',      # Export energy map as npy of yaml file ('numpy' or 'yaml')
                 'export_structures': 1,          # Number of min. energy structures to export
                 'export_format': 'xyz',          # Export structure file format
                 'export_pbc': True,              # Export coordinates after applying PBC
```

Comment:
This commit is well-organized, makes sense, and the commit message tracks. This will be super easy to review in code review!